packages/create-starter/src/index.ts

```diff
@@ -74,30 +74,43 @@ export async function main() {
   }
 
   try {
-    const packageJSON = JSON.parse(await fs.readFile(path.join(destPath, 'package.json'), 'utf8'));
-    packageJSON.name = options.name;
-    packageJSON.version = '0.1.0';
-    EXCLUDED_PACKAGE_JSON_FIELDS.forEach((field) => delete packageJSON[field]);
-
-    try {
-      await fs.writeFile(path.join(destPath, 'package.json'), JSON.stringify(packageJSON, null, 2));
-      await overrideAngularJsonIfExists(destPath, options.kit, options.name);
-
-      await initGitRepo(destPath);
-    } catch (_) {
-      console.info(gray(`> ${bold('Note:')} Failed to update package.json. You may need to do this manually.`));
+    const packageJsonPath = path.join(destPath, 'package.json');
```

Comment:
@danecando, this is what we talked about a few days ago. I made this adjustment for non-Node starter kits. Does this look OK to you?